src/expressions/functions/builtInFunctions.sequences.ts

```diff
@@ -299,6 +300,7 @@ const fnIndexOf: FunctionDefinitionType = (
 ) => {
 	return search.mapAll(([onlySearchValue]) =>
 		sequence
+			.atomize(_executionParameters)
```

Comment:
Is this ok?

Reply:
That's ok, but usually we only use the underscore-prefixed variable names for unused variables. Since you're using it, I'd remove the underscore.

Reply:
ok, I'm going to change it!